docs/about.md

```diff
@@ -27,34 +28,49 @@ description: Additional info about this project
 - [**Revolt**](https://github.com/revoltchat/api): open source user-first chat platform
 - [**Spacebar**](https://github.com/spacebarchat): a free, open source, self-hostable Discord-compatible chat/voice/video platform
 - [**Supabase**](https://github.com/supabase/supabase): The open source Firebase alternative.
-- [**Twitter API**](https://github.com/twitterdev/twitter-api-typescript-sdk): Official SDK for the Twitter API
 
 ## Project goals
 
+### openapi-typescript
+
 1. Support converting any valid OpenAPI schema to TypeScript types, no matter how complicated.
 1. Generated types should be statically-analyzable and runtime-free (with minor exceptions like [enums](https://www.typescriptlang.org/docs/handbook/enums.html).
 1. Generated types should match your original schema as closely as possible, preserving original capitalization, etc.
 1. Typegen only needs Node.js to run (no Java, Python, etc.) and works in any environment.
 1. Support fetching OpenAPI schemas from files as well as local and remote servers.
 
-## Differences
```

Comment:
Note: the differences are removed; I debate whether or not these are useful. If we think they are, they should probably go in the intro?

Reply:
I think for folks evaluating openapi-typescript, this can be very useful (and IMO having reasonable differences listed is usually a sign for the health of the project).

Appending them to the end of "Introduction" or even leaving them here in an appropriate subsection seem fine to me.

Looking at the other edits, it does make me wonder if we should rename "About" to "Project Goals". It feels to me that both the goals and the differences have a better home under the individual headers.

(We might want to consider a overall about page with a higher level goal, like "We take OpenAPIs and make useful typescript stuff with it", of course, we need to first agree on that one :P).

In the interest of time, I have not reviewed below this comment yet. Let's settle on overall structure first.